### DISCUSSION TITLE
Wikipedia style template?

### DISCUSSION THREAD

**Discussion by raghu-mahajan** [executablebooks/meta#1161] [Q&A] [answered]
I am a researcher in theoretical physics trying to build a website using MyST for aggregating the know how that is important for my research group. 

I have explored the MyST book theme in the past few months, but I have realized that I would like to structure my website in a different format. Roughly, I want it to be a encyclopedia, without any ordered table of contents. Instead, I would like a landing page with a search bar, much like Wikipedia, say. 

Does executablebooks have support for such a website?

**Comment by rowanc1** [comment, marked as answer]
We don't have that style of theme out of the box, no. Right now there are two supported themes: the `article-theme` and the `book-theme`. However, there are also ways to create completely custom themes, but they do require knowledge of react. If that is something that you are interested in we can provide improved documentation for how to get started on your own.

**Reply by raghu-mahajan** [reply]
Thank you for the quick response. Unfortunately, right now, I don't have the bandwidth for learning react, but perhaps at some point in the future.

**Reply by choldgraf** [reply]
@raghu-mahajan I'd also recommend giving a +1 or voicing support for this issue!

https://github.com/executablebooks/mystmd/issues/100

**Comment by retreaduntamed** [comment]
@rowanc1 Could you still publish additional documentation on creating custom site themes for those with react knowledge? Unfortunately there isn't much to go on [here](https://github.com/myst-templates/book-theme). Appreciate the work on this project.
> We don't have that style of theme out of the box, no. Right now there are two supported themes: the `article-theme` and the `book-theme`. However, there are also ways to create completely custom themes, but they do require knowledge of react. If that is something that you are interested in we can provide improved documentation for how to get started on your own.
